skunkworks_crow/src/main/java/org/odk/share/views/ui/about/AboutActivity.java

```diff
@@ -47,19 +51,25 @@ protected void onCreate(Bundle savedInstanceState) {
         llm.setOrientation(RecyclerView.VERTICAL);
         recyclerView.setLayoutManager(llm);
         adapter = new AboutAdapter(this, this);
+        adapter.addItem(new AboutItem(R.string.app_information, R.drawable.ic_stars));
         adapter.addItem(new AboutItem(R.string.open_source_licenses, R.drawable.ic_stars));
         adapter.addItem(new AboutItem(R.string.user_guide, R.drawable.ic_stars));
         recyclerView.setAdapter(adapter);
     }
 
-    @Override
+            @Override
     public void onItemClick(View view, int position) {
         if (position == 0) {
+
+            new CustomTabsIntent.Builder()
```

Comment:
My main concern was that what if the device doesn't support custom tabs, it would be easier for you to understand if you can explore ODK-Collect Custom tabs implementation that they have because that handles lot of things which we might not be able to test on our devices, there are even few devices which doesn't have browser in them, so this change may crash the app on those devices.

Reply:
@lakshyagupta21 Should i make changes according to odk-collect custom tabs?